blog/2025-04-22-bucket-domain-names/index.mdx

```diff
@@ -0,0 +1,99 @@
+---
+slug: bucket-domain-names
+title: "Decoupling Bucket Names and Domain Names"
+description: >
+  Tigris lets you set any domain you want to point to your buckets instead of
+  making the domain name match the bucket name.
+image: ./bucket-label.jpg
+keywords: [object storage, blob storage, s3]
+authors: [ks]
+tags: [feature]
+---
+
+import InlineCta from "@site/src/components/InlineCta";
+
+On Tigris, you can set any domain name to point to any bucket. Why does this
+matter?
+
+Multi-cloud is all the rage these days. A new neocloud pops up every week
+offering cheap compute, and running your apps on Kubernetes means they’re easily
+ported to another provider. Compute is fungible, but storage is sticky. Using a
+custom domain to front your bucket of data makes it easy to swap out storage
+providers without updating all the code that uses that storage.
+
+![](./bucket-label.jpg)
+
+{/* truncate */}
+
+Your domain is your brand identity on the Internet. People want to know you as
+you, not as something or someone else. If your downloads come from
+yourcompany.com, that looks like they have _much more authority_ than if they
+come from yourcompany-public-downloads.someprovider-storage.dev. If you are
+operating in environments that require you to explicitly allowlist egress
+domains, then many times you don’t need to do anything because your own domain
+surely is already on that allow list. This makes it easy to glance at a domain
+name and get a good vibe as to what it’s there for.
+
+Oh, not to mention, you own your domain name. If you want to move to another
```

Comment:
Just changing the domain name -> do you mean CNAMing it to a different provider? Even in that case though not everything would be migrated over until the data is migrated, no?